pkg/querier/querier.go

```diff
@@ -553,7 +553,7 @@ func (querier) Close() error {
 	return nil
 }
 
-func (q querier) mergeSeriesSets(sets []storage.SeriesSet) storage.SeriesSet {
+func (q querier) mergeSeriesSets(ctx context.Context, sets []storage.SeriesSet) storage.SeriesSet {
```

Comment:
I have another pr that removes this function totally https://github.com/cortexproject/cortex/pull/5825 so might affect this change.

Reply:
removed the change. It will be removed with above PR anyway.